src/course-unit/add-component/messages.js

```diff
@@ -0,0 +1,14 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  title: {
+    id: 'course-authoring.course-unit.add.component.title',
```

Comment:
[important] You added translations, but I don't see any changes in the key files (`i18n` directory). 
Please add 🥇

Reply:
Added